Makefile

```diff
@@ -113,6 +114,10 @@ test: all
 	pytest test/ -v
 
 
+download-fonts:
+	svn export https://github.com/matplotlib/matplotlib/trunk/lib/matplotlib/mpl-data/fonts/ttf
+	mv ttf build/fonts
```

Comment:
You could probably just copy them from `packages/matplotlib/build/matplotlib-2.2.3/lib/matplotlib/mpl-data/fonts/`, and put `build/packages.json` as a dependency on this task so it will run after the packages get built.

Reply:
Done :)